hrui/hrui.php

```diff
@@ -921,7 +921,9 @@ function _hrui_coreMenuChanges(&$params) {
 
   $civiCaseNavId = CRM_Core_DAO::getFieldValue('CRM_Core_DAO_Navigation', 'CiviCase', 'id', 'name');
   $redactionRulesNavId = CRM_Core_DAO::getFieldValue('CRM_Core_DAO_Navigation', 'Redaction Rules', 'id', 'name');
+  $supportNavId = CRM_Core_DAO::getFieldValue('CRM_Core_DAO_Navigation', 'Support', 'id', 'name');
```

Comment:
Nice, I thought you'd have to loop through each $param and check the name, but this is better.